internal/config/validation.go

```diff
@@ -17,12 +18,42 @@ import (
 	"github.com/edgelesssys/constellation/v2/internal/versions"
 	ut "github.com/go-playground/universal-translator"
 	"github.com/go-playground/validator/v10"
+	"golang.org/x/mod/semver"
 )
 
 func validateK8sVersion(fl validator.FieldLevel) bool {
 	return versions.IsSupportedK8sVersion(fl.Field().String())
 }
 
+func registerInvalidK8sVersionError(ut ut.Translator) error {
+	return ut.Add("invalid_k8s_version", "{0} specifies an unsupported Kubernetes version. {1}", true)
+}
+
+func translateInvalidK8sVersionError(ut ut.Translator, fe validator.FieldError) string {
+	validVersions := make([]string, len(versions.VersionConfigs))
+	i := 0
+	for k := range versions.VersionConfigs {
+		validVersions[i] = string(k)
+		i++
+	}
+	validVersionsSorted := semver.ByVersion(validVersions)
+	sort.Sort(validVersionsSorted)
+
+	var errorMsg string
+	configured := fe.Value().(string)
```

Comment:
```suggestion
	configured, ok := fe.Value().(string)
	if !ok {
		errorMsg = "The configured version is not a valid string."
	}
```
Not sure about how to handle an invalid cast, but I think we should handle it somehow.